code/def_files/data/effects/post-f.sdr

```diff
@@ -18,6 +18,14 @@ layout (std140) uniform genericData {
 
 	vec3 tint;
 	float dither;
+
+	// these are blank, valid slots for modders to create custom effects 
+	// that can be defined in post_processing.tbl and coded below
+	//float custom_effect_float_a;
+	//float custom_effect_float_b;
+
+	//vec3 custom_effect_vec3_a;
+	//vec3 custom_effect_vec3_b;
```

Comment:
These should not be commented out.
I don't know if all drivers correctly react to the bound uniform buffer having more elements than the one in the shader.
Also, these need to be interleaved vec3_a, float_a, vec3_b, float_b. vec3's need to start 4-byte-aligned.